include/wlr/wayland.h

```diff
@@ -10,8 +10,8 @@ struct wlr_output_mode_state;
 struct wlr_output_mode {
 	struct wlr_output_mode_state *state;
 	uint32_t flags; // enum wl_output_mode
-	int32_t width, height;
-	int32_t refresh; // mHz
+	uint32_t width, height;
+	uint32_t refresh; // mHz
```

Comment:
This is actually taken straight from the wayland protocol, which uses signed integers. Why? I have no clue.